packages/esm-ward-app/src/ward-patient-card/ward-patient-card.tsx

```diff
@@ -13,6 +15,14 @@ const WardPatientCard: React.FC<WardPatientCardProps> = (props) => {
       {patientCardRows.map((WardPatientCardRow, i) => (
         <WardPatientCardRow key={i} {...props} />
       ))}
+      <button
+        className={styles.wardPatientCardButton}
+        onClick={() =>
+          launchWorkspace<WardPatientWorkspaceProps>('ward-patient-workspace', { patientUuid: props.patient.uuid })
+        }>
+        {/* Name will not be displayed; just there for a11y */}
```

Comment:
Not sure what this means, but ok

Reply:
a11y = accessibility. The name serves as a label for the box/button so that the browser knows what it is.